packages/kbn-chart-icons/package.json

```diff
@@ -2,5 +2,6 @@
   "name": "@kbn/chart-icons",
   "private": true,
   "version": "1.0.0",
-  "license": "SSPL-1.0 OR Elastic License 2.0"
+  "license": "SSPL-1.0 OR Elastic License 2.0",
+  "sideEffects": false
```

Comment:
What this sideEffects do?

Reply:
It is a way to tell webpack (or in general to a bundler) that the package has no side effect and can be treeshaken: https://webpack.js.org/guides/tree-shaking/#conclusion